.github/workflows/percy.yml

```diff
@@ -30,18 +22,18 @@ jobs:
 
     steps:
       - name: Bail early
-        if: github.event_name == 'pull_request' && ((contains(github.event.pull_request.labels.*.name, 'skip:ci') || contains(github.event.pull_request.labels.*.name, 'skip:percy')) || (github.event.pull_request.draft == true))
         run: exit 0
+        if: github.event_name == 'pull_request' || github.actor == 'dependabot[bot]'
 
       - name: Clone repository
         uses: actions/checkout@v2
-        if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.pull_request.draft == false && !contains(github.event.pull_request.labels.*.name, 'skip:ci') && !contains(github.event.pull_request.labels.*.name, 'skip:percy'))
```

Comment:
We can remove `skip:percy` and `skip:browserstack` from everywhere if we can only launch those jobs manually or automatically on 2 branches.